packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -395,6 +398,39 @@ async function pay(
   }
 }
 
+export function resolveIlpDestination(receiver: Receiver): Pay.ResolvedPayment {
+  const ilpPaymentMethod = receiver.paymentMethods.find(
+    (method) => method.type === 'ilp'
+  )
+
+  if (!ilpPaymentMethod) {
+    throw new PaymentMethodHandlerError(
+      'Invalid ILP payment method on receiver',
+      {
+        description: 'No ILP payment method found in receiver',
+        retryable: false
+      }
+    )
+  }
+
+  if (!isValidIlpAddress(ilpPaymentMethod.ilpAddress)) {
+    throw new PaymentMethodHandlerError(
+      'Invalid ILP payment method on receiver',
+      {
+        description: 'Invalid ILP address for ILP payment method',
+        retryable: false
+      }
+    )
+  }
+
+  return {
+    destinationAddress: ilpPaymentMethod.ilpAddress as IlpAddress,
+    destinationAsset: receiver.asset,
+    sharedSecret: base64url.toBuffer(ilpPaymentMethod.sharedSecret),
+    requestCounter: Pay.Counter.from(0) as Pay.Counter
+  }
+}
```

Comment:
The logic that lived previously in the receiver, https://github.com/interledger/rafiki/pull/3460/files#diff-6c83fb3a86095e274a059aa3299a1b2f4ad0560722d661de537e7b8296c34e71

now we just get ilpDetails from the payment methods, when the caller needs it, like in this ilpPaymentService